iiab-network

```diff
@@ -85,6 +88,9 @@ if [[ `command -v nmcli` ]]; then
     echo
     nmcli c
 fi
+if [[ `command -v networkctl` ]]; then
```

Comment:
```suggestion
if command -v networkctl > /dev/null; then
```

Reply:
ditto, and check for formatting, might need an echo or two